File: Revision_classes/09_1_SM&NSM/Test7.java
class A
{
	static int a = 10;
	static {
		System.out.println("Hi!");
	}
}
class B extends A
{
	static int b = 20;
	static {
		System.out.println("Hello!");
	}
}
class  Test7
{
	public static void main(String[] args) 
	{
		System.out.println(new B().a);
		
	}
}

File: React/app7/src/Test7.js
export default function Test7() {
    const test = () => {
        // Synchronous
        console.log("Start");
        // Synchronous
        const promise = new Promise((resolve, reject) => {
            console.log("Promise executor");
            resolve("Resolved");
        });
        // Microtask
        promise.then((value) => {
            console.log("Promise then: " + value);
        });
        // Macrotask
        setTimeout(() => {
            console.log("setTimeout");
        }, 0);
        // Synchronous
        console.log("End");
    }
    return (
        <>
            <button type="button" onClick={test}>test</button>
        </>
    );
}